openstack_controller/README.md

```diff
@@ -35,11 +35,29 @@ The openstack_controller integration is designed to collect information from all
 
 ### Metrics
 
-Openstack_controller does not include any metrics.
+See [metadata.csv][6] for a list of metrics provided by this integration.
 
 ### Service Checks
+**openstack.neutron.api.up**
+
+Returns `CRITICAL` if the Agent is unable to query the Neutron API, `UNKNOWN` if there is an issue with the Keystone API. Returns `OK` otherwise.
+
+**openstack.nova.api.up**
+
+Returns `CRITICAL` if the Agent is unable to query the Nova API, `UNKNOWN` if there is an issue with the Keystone API. Returns `OK` otherwise.
+
+**openstack.keystone.api.up**
+
+Returns `CRITICAL` if the Agent is unable to query the Keystone API. Returns `OK` otherwise.
+
+**openstack.nova.hypervisor.up**
+
+Returns `UNKNOWN` if the Agent is unable to get the Hypervisor state, `CRITICAL` if the Hypervisor is down. Returns `OK` otherwise.
+
+**openstack.neutron.network.up**
+
+Returns `UNKNOWN` if the Agent is unable to get the Network state, `CRITICAL` if the Network is down. Returns `OK` otherwise.
```

Comment:
I think `openstack.neutron.network.up` can only be `CRITICAL` or `OK`, but not `UNKNOWN`

Reply:
Ok, I copied it from the other openstack_integration